v1/topdown/eval.go

```diff
@@ -1786,7 +1905,7 @@ func (e evalBuiltin) eval(iter unifyIterator) error {
 			case e.bi.Decl.Result() == nil:
 				return iter()
 			case len(operands) == numDeclArgs:
-				if v.Compare(ast.Boolean(false)) == 0 {
+				if ast.Boolean(false).Equal(v) {
```

Comment:
What are these changes about? Anything more than cosmetics? 🤔

Reply:
Don't remember if there was any difference in perf, but I doubt it.. so yeah, mostly cosmetic as the Equal method seemed like the best choice for expressing... equality? :) And putting the static value on the left-hand side ensures nillness returns false instead of a panic, but that's obviously not been a problem here before.